src/main/java/frc/trigon/robot/subsystems/swerve/SwerveModule.java

```diff
@@ -155,15 +172,17 @@ private void configureHardware(double offsetRotations) {
     }
 
     private void configureSignals() {
-        driveMotor.registerThreadedSignal(TalonFXSignal.POSITION, PoseEstimatorConstants.ODOMETRY_FREQUENCY_HERTZ);
+        steerMotor.registerThreadedSignal(TalonFXSignal.POSITION, PoseEstimatorConstants.ODOMETRY_FREQUENCY_HERTZ);
+        steerMotor.registerSignal(TalonFXSignal.VELOCITY, PoseEstimatorConstants.ODOMETRY_FREQUENCY_HERTZ);
+
```

Comment:
1. no need for the whitespace.
2. mention the drive motor, not steer. 
3. there shouldn't be any change here from main.